src/programs/ectrans-benchmark.F90

```diff
@@ -652,12 +655,17 @@ program ectrans_benchmark
   ! While in grid point space, dump the values to disk, for debugging only
   !=================================================================================================
 
-  if (ldump_values) then
-    ! dump a field to a binary file
-    call dump_gridpoint_field(jstep, myproc, nproma, ngpblks, zgp2(:,1,:),         'S', noutdump)
-    call dump_gridpoint_field(jstep, myproc, nproma, ngpblks, zgpuv(:,nflevg,1,:), 'U', noutdump)
-    call dump_gridpoint_field(jstep, myproc, nproma, ngpblks, zgpuv(:,nflevg,2,:), 'V', noutdump)
-    call dump_gridpoint_field(jstep, myproc, nproma, ngpblks, zgp3a(:,nflevg,1,:), 'T', noutdump)
+  if (ldump_values .and. mod(jstep,10) == 1) then
+    if (myproc == 1) then
+      allocate(global_field(ngptotg,1))
+    endif
+    call dump_gridpoint_field(jstep, myproc, nproma, global_field, zgp2(:,1:1,:), 's', noutdump)
```

Comment:
In my [benchmark refactor PR](https://github.com/ecmwf-ifs/ectrans/pull/122/files#diff-bfd9d146dfab7708c079d896a09f7c52c70fa17572bd2c12fddaf6316ff93427R636), I decided to only dump one field. All fields basically go through the same code path so I don't think it's necessary to dump every field. What do you think?

Reply:
I want dump_gridpoint_field to take a 3d array, such that it is easy to change when there is need, but I agree that it is a good default to only dump one per field. Let me change to default last level only

Reply:
=> done. can you agree with this solution?

Reply:
oh, I think it makes sense to dump them separately, especiall U/V/T because they go through separate code paths (U/V are transformed into VOR/DIV). Also the errors for U/V behave different compared to T: They increase faster in general.

Reply:
Ah yes, good point. U/V are derived from vor/div so go through a different procedure.